CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 DEPRECATIONS:
 * resource_sumologic_ingest_budget : Deprecated in favour of `resource_sumologic_ingest_budget_v2`.
 
+## 2.31.5 (September 23, 2024)
```

Comment:
Please remove date. Date shall be added on the day of release

Reply:
Sure, done